libsplinter/Cargo.toml

```diff
@@ -17,6 +17,7 @@ zmq-transport = ["zmq"]
 ursa-compat = ["ursa"]
 sawtooth-signing-compat = ["sawtooth-sdk"]
 events = ["hyper", "tokio", "awc"]
+biome = ["diesel"]
```

Comment:
Add an additional feature for "biome-notifications" (biome-notifications should require biome)

Reply:
added